english/lucidream/luciddream.md

```diff
@@ -6,7 +6,7 @@ Its works!! Your breathing through your blocked nose
 confirming yourself your inside a dream...
 what will you do?
 
-Panic and wake up
+[Panic and wake up](wake-up/wake-up.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.